src/pages/Content/index.js

```diff
@@ -26,3 +30,9 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+// The function plays a recording when needed
+function playRecording(recordedEvents) {
+  recordedEvents.forEach(element => {
+    click(element);
```

Comment:
Make sure that the element is found before clicking. Maybe you will need to wait for the element with setainterval

Reply:
Yeah, inside the click function we are checking if the element is found by this: 
function click(selector) {
  const element = document.querySelector(selector) || null;
  if (element) element.click();
}

Do we need to check in the playRecording before passing to click function or do we need to remove that from the click function? Can you please advise?

Reply:
Just to clarify on setinterval. Are you suggesting that we will give little timeout for the clicks before sending the next click? So, by doing so, we will make sure that previous click has enough time to complete before the next one, right?

Reply:
Yes I know it is checked in `click`, but, on this abstraction level I want to make sure the element exists (sometimes it takes time for an element to exist) so we would want to add a `setInterval` to check that it exists before clicking it.

You can open an issue for this so we will add it in a next PR or add the changes here. Let me know what you decide.

Reply:
Great. Maybe better to be in this PR.

Reply:
As discussed, let me create an issue and submit new PR for this.